CHANGELOG.md

```diff
@@ -3,6 +3,26 @@ Datadog Agent 6 has been officially released and the release notes can be found
 Changes
 =======
 
+# 5.32.8 / Unreleased 
+
+**Linux, Windows, Docker and Source Install**
+
+### Changes
+
+* [BUGFIX] Do not always start the profiler in dev mode. See #3888.
+* [BUGFIX] Update deprecated `datadog-cert.pem` certificate. See [#3882][] (Thanks [@sodabrew][]).
+* [BUGFIX] JMXFetch: bump to 0.26.8. Fixes Java6 compatibility + misc fixes. See #3894 and [omnibus-software-354](https://github.com/DataDog/omnibus-software/pull/354).  
+* [CLEANUP] JMXterm: no longer shipped with agent. See [dd-agent-omnibus-399](https://github.com/DataDog/dd-agent-omnibus/pull/399).
```

Comment:
is there a docs page we can link to, that describes how users can download and use jmxterm?

Reply:
Done.